.travis.yml

```diff
@@ -69,6 +69,14 @@ matrix:
       env: TOXENV=py34
     - python: "3.5"
       env: TOXENV=py35
+    - os: osx
+      osx_image: xcode7.3
```

Comment:
What about upgrading OpenSSL through brew though if we really need to run it on the default image?

I'm with running the latest for other reasons, because it's the most commonly used, but if we need the default image maybe we can make it work.